deepchem/models/torch_models/tests/test_hf_models.py

```diff
@@ -178,7 +178,7 @@ def test_load_from_hf_checkpoint():
     from transformers.models.t5 import T5Config, T5Model
     config = T5Config()
     model = T5Model(config)
-    hf_model = HuggingFaceModel(model=model, tokenizer=None, task='regression')
+    hf_model = HuggingFaceModel(model=model, tokenizer=None, task=None)
```

Comment:
A T5Model is used for text classification tasks. I chose it here as the model checkpoint of T5 is one of the smallest and it will save as bandwidth in running the tests.

The plain T5Model outputs raw hidden states when `task == None` and we are testing whether the reloaded model has different weights than new model.

Reply:
Can you add documentation to HFModel about the behavior when `task == None`?

Reply:
added documentation